assets/js/blocks/featured-category/block.js

```diff
@@ -325,7 +308,10 @@ const FeaturedCategory = ( {
 		return (
 			<ResizableBox
 				className={ classes }
-				size={ { height } }
+				size={ {
```

Comment:
I think that we should not limit the height in the editor.
Now the user can change the `font-size`. If the `font-size` is high, the block should become bigger otherwise the user can't see the entire text.